sql_metadata/parser.py

```diff
@@ -30,8 +30,10 @@ class Parser:  # pylint: disable=R0902
     Main class to parse sql query
     """
 
-    def __init__(self, sql: str = "") -> None:
+    def __init__(self, sql: str = "", logging: bool = True) -> None:
         self._logger = logging.getLogger(self.__class__.__name__)
+        if not logging:
+            self._logger.disabled = True
```

Comment:
```suggestion
    def __init__(self, sql: str = "", enable_logging: bool = True) -> None:
        self._logger = logging.getLogger(self.__class__.__name__)
        if not enable_logging:
            self._logger.disabled = True
```

Change the constructor argument name to avoid conflict with the `logging` module.